src/launcher/java/org/truffleruby/launcher/options/CommandLineParser.java

```diff
@@ -559,7 +559,7 @@ private void processEncodingOption(String value) throws CommandLineException {
         }
 
         if (encodings.size() >= 1) {
-            config.setOption(OptionsCatalog.EXTERNAL_ENCODING, encodings.get(1));
+            config.setOption(OptionsCatalog.EXTERNAL_ENCODING, encodings.get(0));
```

Comment:
Ugh.

Reply:
I was curious: https://github.com/oracle/truffleruby/commit/f952a08edb57524d67884f2dead34abe58a74bb8#r20633204 introduced it. And I even noticed, @chrisseaton ;)